platforms/documentation/docs/src/docs/userguide/native/native_software.adoc

```diff
@@ -850,7 +850,7 @@ The core Gradle tool chains are able to target the following architectures out o
 | Architectures
 
 | GCC
-| x86, x86_64, arm64 (macOS Only)
+| x86, x86_64, arm64 (macOS & linux Only)
 
 | Clang
 | x86, x86_64, arm64 (macOS only)
```

Comment:
I believe we treat Clang as a "GCC compatible" toolchain, so these changes apply to it too.

Reply:
done